modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerConfigurationChangesTest.java

```diff
@@ -169,18 +172,20 @@ public void setUp() throws Exception {
 
     @AfterEach
     public void tearDown() throws Exception {
-        distributionZoneManager.stop();
-        metaStorageManager.stop();
-        clusterCfgMgr.stop();
-        vaultMgr.stop();
-        generator.close();
+        IgniteUtils.closeAll(
+                distributionZoneManager == null ? null : distributionZoneManager::stop,
```

Comment:
Why do we need null check here? Do we expect any of closeables to be null in happy case?

Reply:
Since the test can fail at any step, including initialization, checking for `null` looks like the right approach.
Checking/expecting only a happy case doesn't seem right to me.

Reply:
closeAll() internally will handle NPE in order to close all the resources, won't it?

Reply:
You are right, but there is a nuance, it is necessary for the class to implement `AutoCloseable`, none of them do this.

Reply:
Yep, you are right.